deployment_cfn/.specific/overview.adoc

```diff
@@ -1,5 +1,5 @@
-This Quick Start deploys {partner-product-name} on the AWS Cloud. If you are unfamiliar with AWS Quick Starts we recommend that you read the https://aws-ia.github.io/content/qs_info.html[AWS Quick Start General Content Guide].
+This Quick Start deploys {partner-product-name} on the AWS Cloud. If you are unfamiliar with AWS Quick Starts, see the https://aws-ia.github.io/content/qs_info.html[AWS Quick Start General Content Guide^].
```

Comment:
I'd move up into the first paragraph the first line below, which describes the scope of this guide: "This deployment guide covers the steps necessary to deploy this Quick Start." (That line is currently commented out.)

Reply:
Revised. But I kept it where it is because it applies only to operational guides. I also removed the unnecessary comment about scope because we already mention it in the first sentence.